instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/logging/LoggingExporterProperties.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters.logging;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/** Configuration for {@link io.opentelemetry.exporter.logging.LoggingSpanExporter}. */
+@ConfigurationProperties(prefix = "otel.exporter.logging")
```

Comment:
should we open sdk issue to add these as sdk autoconfigure properties?

Reply:
I don't really think so - the spring starter uses a bit different configuration scheme than the SDK (which I dislike, I think they should use exactly the same property names), and the exporters are enabled/disabled in a bit different way.

Reply:
I fully support changing the sprint boot configuration to match sdk configuration if that's possible